packages/core/src/App/Constants/routes-config.js

```diff
@@ -224,32 +257,16 @@ const getModules = ({ is_appstore }, is_social_signup) => {
             getTitle: () => localize('Trader'),
             routes: [
                 {
-                    path: routes.reports,
+                    path: routes.dxtrade,
                     component: Trader,
-                    getTitle: () => localize('Reports'),
-                    icon_component: 'IcReports',
-                    is_authenticated: true,
-                    routes: [
-                        {
-                            path: routes.positions,
-                            component: Trader,
-                            getTitle: () => localize('Open positions'),
-                            icon_component: 'IcOpenPositions',
-                            default: true,
-                        },
-                        {
-                            path: routes.profit,
-                            component: Trader,
-                            getTitle: () => localize('Profit table'),
-                            icon_component: 'IcProfitTable',
-                        },
-                        {
-                            path: routes.statement,
-                            component: Trader,
-                            getTitle: () => localize('Statement'),
-                            icon_component: 'IcStatement',
-                        },
-                    ],
+                    getTitle: () => localize('Deriv X'),
+                    is_authenticated: false,
+                },
+                {
+                    path: routes.mt5,
+                    component: Trader,
+                    getTitle: () => localize('MT5'),
+                    is_authenticated: false,
```

Comment:
MT5 and DerivX are already removed from the trader package. We shouldn't be adding it back